.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+# This workflow will build a golang project
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-go
+
+name: CI
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
```

Comment:
When we backport bug fixes to releases, the pull request would be created against a release branch. Hence, we can use "**" instead of main.